packages/renderer/src/lib/pod/DeployPodToKube.svelte

```diff
@@ -208,7 +239,7 @@ async function deployToKube() {
       {#if kubeDetails}
         <p>Generated pod to deploy to Kubernetes:</p>
         <div class="h-1/3 pt-2">
-          <MonacoEditor content="{kubeDetails}" language="yaml" />
+          <MonacoEditor bind:this="{editor}" content="{kubeDetails}" language="yaml" />
```

Comment:
Updating kubeDetails should update the value inside the MonacoEditor component, no need to get a reference I would say

Reply:
I guess i tried and it didn't work. I'll verify again. But in this case you only set the content, there is no listener that makes the editor refresh when the value is updated

Reply:
Can the MonacoEditor content ever be edited manually? If so we should do a two way binding or it can cause some misbehavior.

Reply:
It's read-only by default so it would be a future issue in case